src/main/scala/org/bitcoins/core/gen/ScriptGenerators.scala

```diff
@@ -223,6 +353,15 @@ trait ScriptGenerators extends BitcoinSLogger {
     //gets rid of Gen[Gen[...]]
     scriptSig.flatMap(g => g)
   }
+
+
+  private def randomSignedScriptSig : Gen[(ScriptSignature, ScriptPubKey, Seq[ECPrivateKey])] = {
```

Comment:
Can this function include a signed `cltvScriptSignature` and `csvScriptSignature`?